lib/upgradedb/postgres.2016011900.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * LMS version 1.11-git
+ *
+ *  (C) Copyright 2001-2016 LMS Developers
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License Version 2 as
+ *  published by the Free Software Foundation.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307,
+ *  USA.
+ *
+ */
+
+$this->BeginTrans();
+
+define('CONFIG_TYPE_MARGINS', 6);
```

Comment:
U mnie na 5.6.16 wszystko zadziałało.

Na pewno trzeba osobno definiować stałe w plikach `upgradedb/*` bo z poziomu tych plików nie ma dostępu do stałych z `lib/definitions.php`.

Jeśli chodzi o błąd to raczej nic złego się nie powinno stać, w najgorszym przypadku `Notice`, którego nie wyświetlamy - https://github.com/lmsgit/lms/blob/master/index.php#L36